Guilherme Bayma

GuiBayma opened the pull request:
Bronze 1.
Adicionado nome do artista.

GuiBayma committed:
Bronze 1. Adicionado nome do artista.

GuiBayma committed:
Bronze 2. Busca implementada.

GuiBayma committed:
Prata 1. Adicionado o  recurso de internacionalizaçāo e é realizada uma busca por filmes e músicas.

GuiBayma committed:
Prata 2. Adicionado podcasts e ebooks em diferentes seções.

GuiBayma committed:
Prata 2. Alterado o header da primeira seçāo para aparecer a palavra "Filme".

GuiBayma committed:
Ouro 2. Realizadas mudanças sugeridas, acrescentado imagens das mídias que são carregadas através de uma URL de forma assíncrona.

ViniciusMiana commented on iTunesSearch/Ebook.h:
```diff
@@ -0,0 +1,17 @@
+
+#import <Foundation/Foundation.h>
+
+@interface Ebook : NSObject
```
Ebook nao eh midia, porque não usa herança?

GuiBayma committed:
Prata 3: Adicionada a NavigationBar e a view de detalhes das midias.
Alterações pedidas pelo professor realizadas: entidades herdam suas proprierties da classe Midia.
Limitadas o numero de midias para 15, de cada tipo.
Adicionadas proprierties nas mídias.

GuiBayma committed:
Adicionado método que utiliza Regular Expression.

GuiBayma committed:
Ouro 1. Persistência de dados da ultima pesquisa realizada, utilizando NSUserDefaults.

GuiBayma committed:
Ouro 3. Incluído uma UIImage para cada tipo de mídia.
